README.md

```diff
@@ -7,17 +7,17 @@
 
 >The best way to create macros.
 
-MacroCompiler compiles from the language [EventMacro](http://openkore.com/index.php/EventMacro) to [OpenKore](https://github.com/OpenKore/openkore/) plugin (that is, Perl). EventMacro is a language to automate the actions from OpenKore – the bot used in the Rangarok online game. It compiles to Perl since OpenKore itself is written in Perl.
+MacroCompiler compiles [EventMacro](http://openkore.com/index.php/EventMacro) to the [OpenKore](https://github.com/OpenKore/openkore/) plugin (that is, Perl). EventMacro is a language to automate the actions from OpenKore – the bot used in the Rangarok online game. It compiles to Perl since OpenKore itself is written in Perl.
```

Comment:
Maybe is better to use `to a OpenKore plugin`, because OpenKore can has many plugins, and MacroCompiler will create a plugin.